ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -70,6 +70,11 @@ Additional_Build_Tools_CentOS7:
 Additional_Build_Tools_x86_64:
   - zulu-9
 
+C_Library_32bit_CentOS_x86_64:
+  - glibc.i686
```

Comment:
Can you add a comment to the end of each line stating why its needed?  In this case, I believe the reason is a dependency for being able to execute a 32bit binary.